torch/linalg/__init__.py

```diff
@@ -140,3 +140,51 @@
     >>> LA.norm(m[0, :, :]), LA.norm(m[1, :, :])
     (tensor(3.7417), tensor(11.2250))
 """)
+
+tensorsolve = _add_docstr(_linalg.linalg_tensorsolve, r"""
+linalg.tensorsolve(input, other, dims=None, *, out=None) -> Tensor
+
+Computes a tensor ``x`` such that ``tensordot(input, x, dims=x.ndim) = other``.
+The resulting tensor ``x`` has the shape equal to ``Q``, where ``Q`` is the shape
+of the sub-tensor of `:attr:`input` consisting of the appropriate number of
+its rightmost dimensions such that ``input.shape = other.shape + Q``.
+Before computing ``x``, dimensions of `:attr:`input` can be moved to the right
+using :attr:`dims` to match the shape requirements.
+
+Supports real and, only on the CPU, complex inputs.
```

Comment:
"Supports real" -> "Supports float"

Reply:
Both real and complex numbers are floating-point numbers. What about 
```suggestion
Supports real-valued and, only on the CPU, complex-valued inputs.
```